model/app.py

```diff
@@ -26,15 +27,15 @@
 from model.messages import GeneToReactionsRemote, GeneRequest
 
 
-ORGANISMS = {
-    'iJO1366',
-    'iMM904',
-    'iMM1415',
-    'iNJ661',
-    'iJN746',
-    'e_coli_core',
+ORGANISM_TO_MODEL = {
```

Comment:
As per discussion, changed to uniprot mnemonic Id e.g. http://www.uniprot.org/taxonomy/303 which is legible and commonly used in the field.